divide_and_conquer/mergesort.py

```diff
@@ -1,45 +1,48 @@
-def merge(a, b, m, e):
-    l = a[b : m + 1]  # noqa: E741
-    r = a[m + 1 : e + 1]
-    k = b
+def merge(arr, left, mid, right): 
+    # overall array will divided into 2 array
+    # l contain left portion of array from left to mid of array
+    # r contain right portion of array from mid+1 to till last of array
+    l = arr[left : mid + 1]  # noqa: E741
+    r = arr[mid + 1 : right + 1]
```

Comment:
Please change `l` to `new_left` and `r` to `new_right`.  The # noqa comment can also be removed.  Thanks!

Reply:
Or `left_arr` and `right_arr` might be better.

Reply:
Hello @cclauss sir, I have updated variable name. 
Please go through it.
Thanks!